src/components/NoPage.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function NoPage() {
+  return (
+    <div>
+      PAGE NOT FOUND
+    </div>
+  );
+}
+
+export default NoPage;
```

Comment:
I kindly suggest you remove the NoPage component or make use of it in the Navbar component to avoid keeping unnecessary files in your code.